apps/api/src/app/environments/environment.ts

```diff
@@ -11,11 +11,8 @@ export default {
     secret: process.env.DRIVING_LICENSE_SECRET,
   },
   education: {
-    xroadLicenseServiceId: 'r1/IS-DEV/EDU/10020/MMS-Protected/license-api-v1',
-    emailOptions: {
-      sendFromEmail: 'development@island.is',
-      useTestAccount: true,
-    },
+    xroadLicenseServiceId: 'IS-DEV/EDU/10020/MMS-Protected/license-api-v1',
+    fileDownloadBucket: 'island-is-dev-download-cache-api',
```

Comment:
maybe name this something related to the project, education license something something

Reply:
No this is a general file downloading bucket.